tests/target/enum.rs

```diff
@@ -27,8 +27,10 @@ enum Bar {
 }
 
 enum LongVariants {
-    First(LOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOONG, // comment
-          VARIANT),
+    First(
+        LOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOONG, // comment
+        VARIANT,
+    ),
```

Comment:
Do you mean struct-like variants? I would rather keep them the same as regular structs, but with a longer inner-width heuristic (maybe don't use it at all?) so we prefer one-liners for variants.

In practice most enums tend to either have struct-like or tuple-like variants, but not both, so I'm not too worried about consistency here. I prefer to keep the different kinds of tuples consistent with each other, and the different kinds of structs consistent with each other.